### PR TITLE
Add robotPeriodic and fix feedback in auto

### PR DESCRIPTION
**Pull request by twinters007**
I didn't want to require robotPeriodic since it's not necessary, so the function passes

Fixes: #79

### CODE REVIEW

File: robotpy_ext/autonomous/selector.py [hunk 1]
```diff
@@ -236,7 +237,11 @@ def run(self, control_loop_wait_time=0.020, iter_fn=None, on_exception=None):
             except:
                 on_exception()
             
-            iter_fn()
+            if isinstance(iter_fn, Iterable):
```
Comment (auscompgeek) [inline robotpy_ext/autonomous/selector.py]: Not sure I'm a huge fan of this check. If we want to do this here, we should check for list or (list, tuple).

Comment (twinters007) [reply robotpy_ext/autonomous/selector.py]: 👍 I was sure there was a better way but google got the best of me

File: robotpy_ext/autonomous/selector.py [hunk 2]
```diff
@@ -236,7 +236,11 @@ def run(self, control_loop_wait_time=0.020, iter_fn=None, on_exception=None):
             except:
                 on_exception()
             
-            iter_fn()
+            if type(iter_fn) in (list, tuple):
```
Comment (auscompgeek) [inline robotpy_ext/autonomous/selector.py]: …

please don't do this. use isinstance instead.

Comment (twinters007) [reply robotpy_ext/autonomous/selector.py]: does isinstance take a list?

**Comment by twinters007** [comment]
I don't really see the need. If it was java and it was harder to pass methods around, maybe. But it's so easy in python

**Comment by auscompgeek** [comment]
You make it sound like it's hard to pass around method references in Java nowadays :wink:

**Comment by auscompgeek** [comment]
I did wonder whether we should avoid conditions in hot loops such as this, but I guess it doesn't really matter in this situation; the loop shouldn't be _that_ hot.

**Pull request merged**